iree/compiler/Conversion/LinalgToVector/Passes.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef IREE_COMPILER_CONVERSION_LINALGTOVECTOR_PASSES_H_
+#define IREE_COMPILER_CONVERSION_LINALGTOVECTOR_PASSES_H_
+
+#include "mlir/Pass/Pass.h"
+
+namespace mlir {
+namespace iree_compiler {
+
+/// Creates a pass to vectorize Linalg operations.
+std::unique_ptr<Pass> createVectorizeLinalgOpsPass();
```

Comment:
Could we change the name of the pass to createLoadStoreVectorizationPass(). Easy to confuse that this is going from Linalg -> Vector dialect, which it is not.

Reply:
Done